### ISSUE TITLE
Error is not described to the user in text, if mandatory filed is left blank, Publish issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#828]
**Describe the bug**
Error is not described to the user in text, if mandatory filed is left blank, Publish issue.

**To Reproduce**

Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test2&path=%2FSites%2Ftest2%2F&pathType=page)
Click on any of the site.
Click on "Add New Server".
On "Server Properties Configuration" popup, Server name field is mandatory but if it left blank and clicked on Save button, no error is displayed.

**Expected behavior**
Error should be described to the user in text, if mandatory filed is left blank.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
This is not a bug as the error for  mandatory field is shown at the bottom of the screen. So closing this.
